src/ocean/LibFeatures.d

```diff
@@ -0,0 +1,24 @@
+/*******************************************************************************
+
+    Meta module defining constants that match ocean feature releases allowing
+    downstream libraries to conditionally implement functionality relying on
+    later ocean versions.
+
+    Copyright:
+        Copyright (c) 2009-2016 Sociomantic Labs GmbH.
+        All rights reserved.
+
+    License:
+        Boost Software License Version 1.0. See LICENSE_BOOST.txt for details.
+        Alternatively, this file may be distributed under the terms of the Tango
+        3-Clause BSD License (see LICENSE_BSD.txt for details).
+
+*******************************************************************************/
+
+module ocean.LibFeatures;
+
+const has_features_2_5 = true;
+const has_features_2_4 = true;
```

Comment:
Remember that already released libraries often depend on unmaintained ocean versions (i.e. turtle v5.x.x depends on ocean 2.3.7) because of semver. So it may happen that I will need to check for features from ocean 2.4 when making new turtle feature release in that branch.

One may judge it as unlikely but adding those consts doesn't cost us anything so I don't see any reason to not do so.